orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/MonitorWebhookTask.java

```diff
@@ -127,7 +127,7 @@ public TaskResult execute(StageExecution stage) {
     } catch (HttpStatusCodeException e) {
       var statusCode = e.getStatusCode();
 
-      if (shouldRetry(statusCode, stageData)) {
+      if (shouldRetry(HttpStatus.valueOf(statusCode.value()), stageData)) {
```

Comment:
Can we change shouldRetry to take HttpStatusCode instead?

Reply:
Done in https://github.com/spinnaker/spinnaker/pull/7057/commits/74fe7243eb8eace4e01f5e3e2ea3b4a25542a424 though this changes the type of `statusCode` from HttpStatus to HttpStatusCode in WebhookMonitorResponseStageData too.  I *think* this is fine, but please do test it to make sure that statusCode still looks the same in the execution context.  FWIW, I see
```
"statusCode": "OK",
```
in an execution context pre-spring-boot-3.0.

Reply:
Sounds good.  Please execute an actual pipeline with a webhook stage to doubly confirm.

Reply:
@dbyron-sf tested this case, and it is working fine.